src/command/archive/push/file.h

```diff
@@ -32,6 +32,7 @@ typedef struct ArchivePushFileResult
 // Copy a file from the source to the archive
 ArchivePushFileResult archivePushFile(
     const String *walSource, bool headerCheck, unsigned int pgVersion, uint64_t pgSystemId, const String *archiveFile,
-    CompressType compressType, int compressLevel, const List *const repoList, const StringList *const priorErrorList);
+    CompressType compressType, int compressLevel, const List *const repoList, const StringList *const priorErrorList,
+    bool modeCheck);
 
```

Comment:
Pointers should not be marked as const in the header.

Reply:
cd7c88a0 unmarks pointers as const in header